### DISCUSSION TITLE
aishell数据集训练完了，export出来的的avg_1.jit.pdiparams如何使用呢

### DISCUSSION THREAD

**Discussion by wonder2025** [PaddlePaddle/PaddleSpeech#725] [Q&A]
aishell训练完了， export的avg_1.jit.pdiparams如何使用呢， export做了什么操作呢，模型从原来460M减少成了198兆

**Comment by zh794390558** [comment]
export是模型转静态图，只保存模型结构和参数，其他的东西都不保存了。
具体例子参看 `https://github.com/PaddlePaddle/DeepSpeech/blob/develop/deepspeech/exps/deepspeech2/bin/test_export.py`
